test/index.js

```diff
@@ -756,6 +756,17 @@ describe('object', function() {
 
   });
 
+  describe('gets', function() {
+
+    it('returns a Maybe', function() {
+      var obj = {x: {z: 0}, y: 42};
+      assert(S.gets(['x', 'z'], obj).equals(S.Just(0)));
+      assert(S.gets(['y'], obj).equals(S.Just(42)));
+      assert(S.gets(['z'], obj).equals(S.Nothing()));
+    });
```

Comment:
You should include a test for `[]`. Also, I like to put the simpler tests before the more complex ones, so I'd move the `['x', 'z']` test after the `['y']` and `['z']` ones.